src/main/java/duke/Eevee.java

```diff
@@ -0,0 +1,67 @@
+ * @version 1.0
+ */
+public class Eevee {
+    public void echo(TaskList tasks, Storage storage) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("~Hello I am Eevee.~\n" + "~What can I do for you?~");
+        System.out.println("                                      |\n" +
+                "                                     /|\n" +
+                "                                   ,' |\n" +
+                "                                  .   |\n" +
+                "                                    | |\n" +
+                "                                 ' '| |\n" +
+                "                                / / | |\n" +
+                "       _,.-\"\"--._              / /  | |\n" +
+                "     ,'          `.           j '   ' '\n" +
+                "   ,'              `.         ||   / ,                         ___..--,\n" +
+                "  /                  \\        ' `.'`.-.,-\".  .       _..---\"\"'' __, ,'\n" +
+                " /                    \\        \\` .\"`      `\"'\\   ,'\"_..--''\"\"\"'.'.'\n" +
+                ".                      .      .'-'             \\,' ,'         ,','\n" +
+                "|                      |      ,`               ' .`         .' /\n" +
+                "|                      |     /          ,\"`.  ' `-. _____.-' .'\n" +
+                "'                      |..---.|,\".      | | .  .-'\"\"   __.,-'\n" +
+                " .                   ,'       ||,|      |.' |    |\"\"`'\"\n" +
+                "  `-._   `._.._____  |        || |      `._,'    |\n" +
+                "      `.   .       `\".     ,'\"| \"  `'           ,+.\n" +
+                "        \\  '         |    '   |   .....        .'  `.\n" +
+                "         .'          '     7  \".              ,'     \\\n" +
+                "                   ,'      |    `..        _,'      F\n" +
+                "                  .        |,      `'----''         |\n" +
+                "                  |      ,\"j  /                   | '\n" +
+                "                  `     |  | .                 | `,'\n" +
+                "                   .    |  `.|                 |/\n" +
+                "                    `-..'   ,'                .'\n" +
+                "                            | \\             ,''\n" +
+                "                            |  `,'.      _,' /\n" +
+                "                            |    | ^.  .'   /\n" +
+                "                             `-'.' j` V    /\n" +
+                "                                   |      /\n" +
+                "                                   |     /\n" +
+                "                                   |   ,'\n" +
+                "                                    `\"\"");
+
+        Lister lister = new Lister(tasks, storage);
+
+        while (sc.hasNextLine()) {
+            String command = sc.nextLine();
```

Comment:
From a user experience standpoint, you might want to consider converting your user input command to lower case (using String toLowerCase()) so that even if the user accidentally capitalizes one or two characters in the command, he/she will not have to retype everything again.

For example, without toLowerCase(), 'Todo read book' or 'Bye' will be considered as an invalid command whereas it will be accepted if the command is converted to lower case first. This will be less punishing for the user if he/she accidentally does not type in all lower cases.

This however, is a design decision depending on how strict you want to be with your user's input!

Reply:
That is an excellent idea!! Thank you :)